submissions/UA-girl/task_friends_app/js/app.js

```diff
@@ -0,0 +1,300 @@
+const URL_FETCH = 'https://randomuser.me/api/?results=100&nat=GB';
+const CARDS_PER_PAGE = 10;
+const MAX_AGE = 100;
+const BREAK_POINT_WIDTH = 467;
+const filterSet = document.querySelector('.container-filters');
+const pages = document.querySelector('.pages');
```

Comment:
I'd recommend renaming this variable to the `pagesWrap`. Because now it's confusing a little bit